### PR TITLE
Punjabsso api

### PR DESCRIPTION
**Pull request by rakeshSgr**

### CODE REVIEW

File: generics/services/sunbird.js [hunk 1]
```diff
@@ -222,6 +221,33 @@ const verifyToken = function (token) {
     })
 }
 
+/**
```
Comment (aks30) [inline generics/services/sunbird.js:224]: @rakeshSgr - Make keycloak as one word

File: module/punjabSSO/helper.js [hunk 2]
```diff
@@ -182,40 +184,40 @@ module.exports = class punjabSSOHelper {
...
                         }
 
                     } else {
-                        throw userCreationResponse
+                        throw userCreationResponse.message
                     }
 
                 } else {
-                    throw keyCloakData
+                    throw keyCloakData.message
```
Comment (aks30) [inline module/punjabSSO/helper.js]: @rakeshSgr - Let's follow the generic structure of the helper function.

File: module/shikshalokam/helper.js [hunk 3]
```diff
@@ -1,250 +0,0 @@
...
-const keycloakSessionScope = (process.env.sunbird_keycloak_session_scope && process.env.sunbird_keycloak_session_scope != "") ? process.env.sunbird_keycloak_session_scope : "offline_access"
-const userCreationChannel = (process.env.sunbird_keycloak_default_user_creation_channel && process.env.sunbird_keycloak_default_user_creation_channel != "") ? process.env.sunbird_keycloak_default_user_creation_channel : "SHIKSHALOKAM"
-const userCreationEndpoint = "/api/user/v1/create"
-// let adminAuthToken = ""
-// const keyCloakAdminUserName = (process.env.sunbird_keycloak_admin_username && process.env.sunbird_keycloak_admin_username != "") ? process.env.sunbird_keycloak_admin_username : ""
-// const keyCloakAdminPassword = (process.env.sunbird_keycloak_admin_password && process.env.sunbird_keycloak_admin_password != "") ? process.env.sunbird_keycloak_admin_password : ""
-const shikshalokamBaseHost = (process.env.SUNBIRD_URL && process.env.SUNBIRD_URL != "") ? process.env.SUNBIRD_URL : ""
-
-module.exports = class shikshalokamHelper {
```
Comment (aks30) [inline module/shikshalokam/helper.js]: @rakeshSgr - Please keep this file a backup for some time now.

File: generics/services/sunbird.js (hunk 1 above)
Comment (rakeshSgr) [reply generics/services/sunbird.js:224]: resolved

**Pull request merged**